packages/expo-camera/android/src/main/java/expo/modules/camera/analyzers/BarcodeAnalyzer.kt

```diff
@@ -58,7 +60,7 @@ class BarcodeAnalyzer(private val lensFacing: CameraType, formats: List<BarcodeT
               barcode.displayValue,
               raw,
               extra,
-              cornerPoints,
+              cornerPoints.toMutableList(),
```

Comment:
```suggestion
              cornerPoints,
```
Or even better can we refactor the code to use `IntArray` everywhere instead of passing a list?